### ISSUE TITLE
Flag error on quick actions

### ISSUE THREAD

**Issue by danymat** [nvim-neorg/neorg#126]
Hey @vhyrro, i Noticed a weird bug when pressing a flag. (#108)

1. Open `Neorg gtd quick_actions`
2. Add a task `a`
3. Create the task
4. Add contexts `c`
5. Add contexts
6. Add waiting for (or press any other flag) <-- Does not work (but somehow `<CR>` does work)

`E5108: Error executing lua ...tart/neorg/lua/neorg/modules/core/ui/selection_popup.lua:19: attempt to call a nil value`

**Comment by danymat** [comment]
I just did a little more debugging, and somehow:
(If I have this:) 
```
General informations
c -> Add Contexts
w -> Add Waiting Fors

Dates
d -> +Add a due date
s -> +Add a start date

<CR> -> Finish
```
I can press `s`, then `d`, then `w`, then `c` but can't do the other way

**Comment by danymat** [comment]
Closed on 4b4ecb4
